test/schema/smbgTest.js

```diff
@@ -43,5 +43,16 @@ describe('schema/cbg.js', function(){
     helper.expectUnitConversion(goodObject, 'value');
   });
 
+  describe('subType', function(){
+    var withSubType = _.assign({}, goodObject, {subType: ''});
+    helper.okIfAbsent(goodObject, 'subType');
+    helper.expectStringField(withSubType, 'subType');
```

Comment:
No, the first one checks if it's OK that the whole field is absent. Note that I give it the unmodified `goodObject` there, not the new `withSubType` copy. Could split it into two tests if you like?